packages/webapp/src/components/Customers/CustomersSelect.tsx

```diff
@@ -34,7 +34,7 @@ function CustomerSelectRoot({
     <FSelect
       items={items}
       textAccessor={'display_name'}
-      labelAccessor={'currency_code'}
+      labelAccessor={'balance'}
```

Comment:
What you did is exactly what we want, it would be better if you used `"formatted_balance"` attribute instead of the row value of `balance`,

Reply:
Sure, let me send an update !